.github/resign.go

```diff
@@ -83,14 +86,17 @@ func checkAndSign(tag string, ctx *types.Context) {
 		fulcioFlag = fmt.Sprintf("--fulcio-url=%s", fulcioURL)
 		ctx.Info("Found FULCIO_URL var, setting the following flag to the cosing command:", fulcioFlag)
 	}
-
-	args := []string{"verify", tag}
+	args = []string{"verify", tag}
 	ctx.Debug("Calling cosing with the following args:", args)
 	out, err := exec.Command("cosign", args...).CombinedOutput()
 	ctx.Debug("Verify output:", string(out))
 	if err != nil {
 		ctx.Warning("Artifact", tag, "has no signature, signing it")
-		args := []string{fulcioFlag, "sign", tag}
+		if fulcioFlag != "" {
+			args = []string{"sign", fulcioFlag, tag}
```

Comment:
shall we maybe trim spaces also from `fulcioFlag`? :clown_face:

Reply:
you joke, but I did just in case LOL